lib/Partitioner/Partitioner.cpp

```diff
@@ -45,7 +45,7 @@ static llvm::cl::OptionCategory PartitionerCat("Glow Partitioner Options");
 static llvm::cl::opt<bool>
     logPartition("log-partition",
                  llvm::cl::desc("Enable logging partition info"),
-                 llvm::cl::init(true), llvm::cl::cat(PartitionerCat));
+                 llvm::cl::init(false), llvm::cl::cat(PartitionerCat));
```

Comment:
@beicy this was making it pretty hard to see test outputs so I set it to false, let me know if you want me to change it back to true but it seems like it should default to false to me.